connectors/aliyun/rocketmq-connect-http/README.md

```diff
@@ -0,0 +1,37 @@
+# rocketmq-connect-http
```

Comment:
@zhaohai1299002788 Would you like to add an introduction for this connector？ other connectors also have this problem

Reply:
HTTP, MNS, dingtalk, and FC have all added descriptions